lib/hz2600/CallflowBuilder/Builder.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace CallflowBuilder;
+
+use CallflowBuilder\Node\AbstractNode; 
+
+class Builder {
```

Comment:
Originally builder was intended to add the numbers as well, but we cannot replace the entire body via the SDK so now it just builds flows and you have to use the SDK callflow to assign the numbers to the flow. 

I talked to karl about removing it, but he was thinking it should stay so that it is more apparent to the user what they are doing. You could still just call $whatever_is_the_root_node->build() to accomplish the same thing.